routes/api/contacts.js

```diff
@@ -1,25 +1,87 @@
 const express = require('express')
-
 const router = express.Router()
+const Joi = require('joi')
+const contactSModel = require('./../../models/contacts')
+
+
+
+const contactSchema = Joi.object({
+  name: Joi.string(),
+  email: Joi.string(),
+  phone: Joi.number()
```

Comment:
phone is not a number. It should be string, also joi has better validation for email